command/pipelines/pipeline_execute.go

```diff
@@ -91,7 +101,33 @@ func (c *PipelineExecuteCommand) Run(args []string) int {
 		return 1
 	}
 
-	c.ApiMeta.Ui.Output(c.ApiMeta.Colorize().Color(fmt.Sprintf("[reset][bold][green]Pipeline execution started")))
+	executions := make([]interface{}, 0)
+	attempts := 0
+	for len(executions) == 0 && attempts < 5 {
+		executions, resp, err = c.queryExecution()
+		attempts += 1
+	}
+	if err != nil {
+		c.ApiMeta.Ui.Error(fmt.Sprintf("%s\n", err))
+		return 1
+	}
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		c.ApiMeta.Ui.Error(fmt.Sprintf("Encountered an error querying pipeline execution, status code: %d\n", resp.StatusCode))
+		return 1
+	}
+	if len(executions) == 0 {
+		c.ApiMeta.Ui.Error(fmt.Sprintf("Unable to start any executions, server response was: %v", resp))
+		return 1
+	}
+
+	refIds := make([]string, 0)
+	for _, execution := range executions {
+		refIds = append(refIds, execution.(map[string]interface{})["id"].(string))
+	}
+	c.ApiMeta.Ui.Output(fmt.Sprintf("Started pipeline execution. Pipeline '%s' has %d executions running with ids: %v\n",
```

Comment:
I wonder if we should make this more "programmer friendly" by just emitting the list rather than all the extra text

Reply:
I'll change this to output just the list; however, I think we need further changes adding a `-q` quiet output flag to make this really script-able. For instance alongside the list output I have `Caching oauth2 token.` We could probably remove that specific output, but it may make sense to add levels/output suppression.